src/components/form/PhoneInquiryForm.tsx

```diff
@@ -0,0 +1,117 @@
+import React, { useState } from 'react';
+import { FormProvider, useForm } from 'react-hook-form';
+import { yupResolver } from '@hookform/resolvers/yup';
+import { phoneInquiryValidationSchema } from '../../utils/validationSchema';
+import PersonalInformationSection from './sections/PersonalInformationSection';
+import ResidenceStatusSection from './sections/ResidenceStatusSection';
+import LanguageSection from './sections/LanguageSection';
+import SupportNeedsSection from './sections/SupportNeedsSection';
+import AsylumProcedureSection from './sections/AsylumProcedureSection';
+import MaritalStatusSection from './sections/MaritalStatusSection';
+import ChildrenInformationSection from './sections/ChildrenInformationSection';
+import EmploymentSection from './sections/EmploymentSection';
+import FormNavigation from './FormNavigation';
+
+interface PhoneInquiryFormProps {
+  onSubmit: (data: any) => void;
+}
+
+const defaultValues = {
+  inquiryDate: new Date().toISOString().split('T')[0],
+  lastName: '',
+  firstName: '',
+  aliasName: '',
+  placeOfResidence: '',
+  phoneNumber: '',
+  timeInGermany: '',
+  nationality: '',
+  residenceStatus: '',
+  visaType: '',
+  languages: '',
+  preferredLanguage: '',
+  otherLanguage: '',
+  specificSupportNeeds: '',
+  acuteThreatSituation: false,
+  supportTypes: [] as string[],
+  otherSupportType: '',
+  asylumProcedureStatus: '',
+  asylumProcedureDetails: '',
+  maritalStatus: '',
+  isMarried: false,
+  marriedSince: '',
+  partnerResidenceStatus: '',
+  hasChildrenInGermany: false,
+  numberOfChildren: 0,
+  childrenAges: '',
+  childrenResidenceStatus: '',
+  isWorking: false,
+  workingHours: '',
+};
+
+const PhoneInquiryForm: React.FC<PhoneInquiryFormProps> = ({ onSubmit }) => {
+  const [currentStep, setCurrentStep] = useState(1);
+  const totalSteps = 8;
```

Comment:
Let's use a map for the steps, I want to see in the map directly which form belongs to which step